Crash when running a script via an association

**Describe the bug**
Running a Python script with the new manager installed crashes:

```
❯ xxx
********************************************************************************
To see all available commands, run 'py help'
********************************************************************************
[ERROR] INTERNAL ERROR: LookupError: C:\Users\Gustav\.local\scripts\xxx.py
[ERROR] Internal error 0x00000001. Please report to https://github.com/python/pymanager
```

**To Reproduce**
Steps to reproduce the behavior:
1. I have Python 3.13 installed from the old MSI installer, including the `py` launcher.
2. I have `.py` in my PATHEXT variable, so running a Python file from the command line works.
3. Typing `xxx` runs `xxx.py`, via the shebang line.
4. Install the pymanager application. Do *not* run on first install, and do not install any Python runtimes.
5. Type `xxx` at the command line again. This now causes the crash shown above.

This is reproducible - I can uninstall the manager to get the old behaviour, and reinstall it to get the crash.

**Expected behavior**
The command `xxx` runs `xxx.py`, exactly as under the old `py` launcher.

**Additional context**
I don't have file associations for `.py` files. They appear to have been removed at some point in the various modifications I made to try out the new manager previously. I did "associate" `.py` files wih Python via the explorer GUI, but I don't know where that information is recorded these days (it's clearly not in the registry, see the output of ftype and assoc below).

❯ cmd /c ftype Python.File
File type 'Python.File' not found or no open command associated with it.
❯ cmd /c assoc .py
File association not found for extension .py

Log file:

```
>> Writing logs to C:\Users\Gustav\AppData\Local\Temp\python_install_20250511225707_34284.log
>> Python installation manager 25.0b5
Copyright (c) Python Software Foundation. All Rights Reserved.

BEGIN install_command.execute: []
>  ********************************************************************************
>  To see all available commands, run 'py help'
>  ********************************************************************************
END install_command.execute
!! INTERNAL ERROR: LookupError: C:\Users\Gustav\.local\scripts\xxx.py
TRACEBACK:
Traceback (most recent call last):
  File "manage\__init__.py", line 50, in main
  File "manage\commands.py", line 806, in execute
  File "manage\install_command.py", line 628, in execute
  File "manage\scriptutils.py", line 168, in find_install_from_script
  File "manage\scriptutils.py", line 163, in _read_script
LookupError: C:\Users\Gustav\.local\scripts\xxx.py
```

Note that the script itself has no shebang line. It's a 5-line Python script with no dependencies apart from the stdlib.

--- a/src/manage/install_command.py
+++ b/src/manage/install_command.py
@@ -625,7 +625,10 @@ def execute(cmd):
         if cmd.from_script:
             # Have already checked that we are not using --by-id
             from .scriptutils import find_install_from_script
-            spec = find_install_from_script(cmd, cmd.from_script)
+            try:
+                spec = find_install_from_script(cmd, cmd.from_script)
+            except LookupError:
+                spec = None
             if spec:
                 cmd.tags.append(tag_or_range(spec))
             else:

